src/scrapy_settings_log/__init__.py

```diff
@@ -51,6 +51,16 @@ def spider_closed(self, spider):
         regex = settings.get("SETTINGS_LOGGING_REGEX")
         if regex is not None:
             settings = {k: v for k, v in settings.items() if re.search(regex, k)}
+        if settings.get("MASKED_SENSITIVE_SETTINGS_ENABLED", True):
+            default_regexes = [
+                ".*(?i)(api[\W_]*key).*",  # apikey and possible variations e.g: shub_apikey or SC_APIKEY
+                ".*(?i)(AWS[\W_]*SECRET[\W_]*ACCESS[\W_]*KEY).*",  # AWS_SECRET_ACCESS_KEY and possible variations
```

Comment:
**suggestion:** I'd maybe just use "SECRET" here, or at least remove the "ACCESS" continuation (we use AWS_SECRET_KEY in my project, for example.).